src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -120,6 +126,12 @@ public TracerSettings(IConfigurationSource source)
         /// <seealso cref="ConfigurationKeys.Environment"/>
         public string Environment { get; set; }
 
+        /// <summary>
+        /// Gets or sets the version tag applied to all spans.
+        /// </summary>
+        /// <seealso cref="ConfigurationKeys.Version"/>
+        public string Version { get; set; }
```

Comment:
Can we call this `ServiceVersion` to avoid confusion?

Reply:
That way we have `ServiceName` and `ServiceVersion`.

Reply:
Done in 1c1b222185b9dbeacf6849d99d8d5181dfe888c9